ql/src/java/org/apache/hadoop/hive/ql/exec/MapOperator.java

```diff
@@ -564,7 +564,12 @@ public void process(Writable value) throws HiveException {
           deserialize_error_count.set(deserialize_error_count.get() + 1);
           throw new HiveException("Hive Runtime Error while processing writable " + message, e);
```

Comment:
We log the "message" here which doesn't have row (which is null) but includes "value" input argument which can potentially have sensitive data. Need to put the "message" into debug log.